anndata/_io/specs/methods.py

```diff
@@ -303,10 +306,30 @@ def write_list(f, k, elem, dataset_kwargs=MappingProxyType({})):
 @_REGISTRY.register_write(ZarrGroup, h5py.Dataset, IOSpec("array", "0.2.0"))
 @_REGISTRY.register_write(ZarrGroup, np.ma.MaskedArray, IOSpec("array", "0.2.0"))
 def write_basic(f, k, elem, dataset_kwargs=MappingProxyType({})):
-    """Write methods which underlying library handles nativley."""
+    """Write methods which underlying library handles natively."""
     f.create_dataset(k, data=elem, **dataset_kwargs)
 
 
+@_REGISTRY.register_write(ZarrGroup, da.Array, IOSpec("dask-array", "0.2.0"))
+def write_zarr(f, k, elem, dataset_kwargs=MappingProxyType({})):
+    g: ZarrGroup = f.create_group(k)
+    # TODO: Understand IOSpec stuff
+    da.Array.to_zarr(
```

Comment:
I don't think this is a static method. I believe it should be used like:

```python
elem.to_zarr(...)
# or
da.to_zarr(elem, ...)
```

Reply:
Actually, looking into `da.to_zarr` and `da.to_hdf5` I think you may not want to use them. You might want to go a bit lower level and use the `dset = f.require_dataset(k, ..., **dataset_kwargs)` and `da.store(elem, dset)`